app/serializers/event-topic.js

```diff
@@ -2,6 +2,10 @@ import ApplicationSerializer from 'open-event-frontend/serializers/application';
 
 export default ApplicationSerializer.extend({
   attrs: {
-    subTopics: 'eventSubTopics'
+    type             : 'event-type',
```

Comment:
This is the event topic serializer. Should only have `subTopics: 'event-sub-topics'`